content-src/experiments/email-tabs.yaml

```diff
@@ -0,0 +1,75 @@
+id: 20
+title: 'Email Tabs'
+slug: email-tabs
+is_featured: false
+platforms: ['addon']
+min_release: 62
```

Comment:
Interesting... [**mozilla/email-tabs** /addon/manifest.json.tmpl:16](https://github.com/mozilla/email-tabs/blob/108586b1ec48d67d12cb8bbc12cf75666fa87f2c/addon/manifest.json.tmpl#L16) says the manifest.json `strict_min_version` is "57.0a1", but here we're saying "62".